lib/index.js

```diff
@@ -1 +1 @@
-global.FormData = module.exports = require('form-data')
+globalThis.FormData = module.exports = require('form-data')
```

Comment:
Maybe conditionally depend on form-data and use built in provided by NodeJS itself?

```diff
- globalThis.FormData = module.exports = require('form-data')
+ globalThis.FormData = module.exports = globalThis.FormData || require('form-data')
```
this could definitely be breaking on the backend as it would no longer have those `form-data` specific signature to eg get the content length/type headers